src/Behat/Config/Formatter/PrettyFormatter.php

```diff
@@ -24,26 +24,25 @@ final class PrettyFormatter extends Formatter
      * @param bool $multiline print out PyStrings and TableNodes in full
      * @param ShowOutputOption $showOutput show the test stdout output as part of the
      *                                     formatter output (yes, no, on-fail)
+     * @param bool $shortSummary if we should print the short summary which just lists scenarios
+     *                           or the long summary which lists steps
      */
     public function __construct(
         bool $timer = true,
         bool $expand = false,
         bool $paths = true,
         bool $multiline = true,
         ShowOutputOption $showOutput = ShowOutputOption::Yes,
+        bool $shortSummary = true,
         ...$baseOptions,
     ) {
-        if ($showOutput === ShowOutputOption::InSummary) {
```

Comment:
The pretty formatter can now show step output in its summary if the short summary option is set to false, so now this option is valid for this formatter and we can remove this check